### DISCUSSION TITLE
Logging variables documentation

### DISCUSSION THREAD

**Discussion by nbelles** [bitcraze/discussions#245] [Q&A]
Hello,

Is there a place to find all the variables that can be logged from the crazyflie? I have been able to find a couple by looking at some of the examples but I would love to be able to see everything that is accessible. 

Thanks!

**Comment by knmcguire** [comment]
Yes there is! Here is automated generated documentation from the source code of the crazyflie firmware: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/api/logs/. So this are all the logs that are present in the current development branch (master) of the crazyflie

Also if you connect to a crazyflie in the [cfclient](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/) you can also open the log toc tab by going to 'view'>'tabs'>'Log TOC', then you can see all the logs that are currently enabled the firmware of your specific crazyflie
